### PR TITLE
Support only current Python versions.

### PR DESCRIPTION
**Pull request by sallner**
It would be nice to have a release after the PR got merged.

### CODE REVIEW

**Comment by sallner** [comment]
@hannosch What do you think about this PR and a release?

File: pythongettext/tests/test_compile.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
 # -*- coding: utf-8 -*-
 import os
+import unittest
 
 from pythongettext.msgfmt import Msgfmt
 from pythongettext.msgfmt import PoSyntaxError
-try:
-    import unittest2 as unittest
```
[the remark this answers is not in the document]
Comment (sallner) [reply pythongettext/tests/test_compile.py]: That is true, done.

**Comment by sallner** [comment]
I released a 4.0 https://pypi.org/project/python-gettext/4.0/

**Comment by hannosch** [comment]
Thanks!

**Pull request merged**